site/access-control.xml

```diff
@@ -297,18 +337,42 @@ limitations under the License.
         This will check LDAP first, and then fall back to the internal
         database if the user cannot be authenticated through LDAP:
       </p>
+
+<pre class="example">
+auth_backends.1 = ldap
+auth_backends.2 = internal
+</pre>
+
+          Or, in the <a href="/configure.html#erlang-term-config-file">classic config format</a>:
+
       <pre class="example">
 [{rabbit, [
             {auth_backends, [rabbit_auth_backend_ldap, rabbit_auth_backend_internal]}
           ]
 }].
       </pre>
-      
+
       <p>
         Same as above but will fall back to the <a
         href="https://github.com/rabbitmq/rabbitmq-auth-backend-http">HTTP backend</a>
         instead:
       </p>
+
+<pre class="example">
+# rabbitmq.conf
+#
+auth_backends.1 = ldap
+# uses module name instead of a short alias, "http"
+auth_backends.2 = rabbit_auth_backend_http
+
+# See HTTP backend docs for details
+http.user_path = http://my-authenticator-app/auth/user
+http.vhost_path = http://my-authenticator-app/auth/vhost
+http.resource_path = http://my-authenticator-app/auth/resource
```

Comment:
Unfortunately if we go with module names in some places but not others, it will be more confusing than using just module names. Which is what we will do now.